src/ProjectSystemTools/BuildLogging/Model/Backend/EvaluationLogger.cs

```diff
@@ -5,8 +5,9 @@
 using System.IO;
 using Microsoft.Build.Framework;
 using Microsoft.Build.Logging;
+using Microsoft.VisualStudio.ProjectSystem.Tools.Providers;
 
-namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.Backend
```

Comment:
"Back End" are separate words, so this would be capitalised as `BackEnd`. Similarly `BackEndBuildTableDataSource`.

Similarly `FrontEnd`.